ale/rotation.py

```diff
@@ -77,7 +83,7 @@ def __mul__(self, other):
                 Another rotation object, it can be constant or time dependent.
         """
         if self.source != other.dest:
-            raise ValueError("Destination frame of first rotation is not the same as source frame of second rotation.")
+            raise ValueError("Destination frame of first rotation {} is not the same as source frame of second rotation {}.".format(other.dest, self.source))
```

Comment:
I have really been liking the new f'{other.dest} blah, blah {self.source}' notation.